dramatiq/brokers/rabbitmq.py

```diff
@@ -344,6 +348,11 @@ def enqueue(self, message, *, delay=None):
                 # next caller/attempt may initiate new ones of each.
                 del self.connection
 
+                if isinstance(e, ChannelClosedByBroker) and e.reply_code == 404 and \
+                        e.reply_text.startswith("NOT_FOUND - no queue"):
```

Comment:
Is the string matching necessary here or can we rely on the code alone?

Reply:
the code is enough